examples/fluids/qfunctions/newtonian.h

```diff
@@ -249,6 +250,16 @@ CEED_QFUNCTION_HELPER int IFunction_Newtonian(void *ctx, CeedInt Q, const CeedSc
     UnpackState_U(s_dot.U, U_dot);
 
     for (CeedInt j = 0; j < 5; j++) v[j][i] = wdetJ * (U_dot[j] - body_force[j]);
+    if (context->idl_enable) {
+      const CeedScalar sigma     = LinearRampCoefficient(1 / context->idl_decay_time, context->idl_length, context->idl_start, x_i[0]);
+      const CeedScalar damp_Y[5] = {sigma * (s.Y.pressure - P0), 0, 0, 0, 0};
+      CeedScalar       dx_i[3]   = {0};
+      State            damp_s    = StateFromY_fwd(context, s, damp_Y, x_i, dx_i);
+      v[0][i] += wdetJ * damp_s.U.density;
+      for (int j = 0; j < 3; j++) v[j + 1][i] += wdetJ * damp_s.U.momentum[j];
+      v[4][i] += wdetJ * damp_s.U.E_total;
+    }
+
```

Comment:
Thoughts on moving this (and it's jacobian) into a separate `CeedOperator`? The obvious concern is with performance, but I'm not sure if there's any rules-of-thumb that tamp down those concerns. 

There aren't any intermediate terms used from the `IFunction_Newtonian` other than state `s`. Possibly we could have the separate IDL CeedOperator use the `jac_data` to accelerate that, though I'm not sure if Ceed guarantees order of execution for sub-operators.

Reply:
I'd rather have fewer operators. I don't think this is expensive, though we could do static specialization if we were concerned about the `if` statement.